### PR TITLE
fix(style): remove allocated field from lv_style_t

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

to simplify the code logic and extend prop_cnt to 256

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update CHANGELOG.md
- [ ] Update the documentation

### CODE REVIEW

File: src/lv_misc/lv_style.c [hunk 1]
```diff
@@ -90,13 +90,20 @@ bool lv_style_remove_prop(lv_style_t * style, lv_style_prop_t prop)
         if(props[i] == prop) {
             lv_style_value_t * values = (lv_style_value_t *)style->v_p.values_and_props;
```
Comment (kisvegabor) [inline src/lv_misc/lv_style.c]: It's not related to your changes but it seems  `old_value` would be a better name for this.

**Review by kisvegabor — COMMENTED** [review]
Looks good!

It is great that `prop_cnt` is the whole byte now because it's read a lot and this way no masking required.

File: src/lv_misc/lv_style.c (hunk 1 above)
Comment (xiaoxiang781216) [reply src/lv_misc/lv_style.c]: Done.

**Review by kisvegabor — APPROVED** [review]
Thanks!

**Pull request merged**